
#### [Guilded.NET.Base](Guilded_NET_Base 'Guilded.NET.Base')
### [Guilded.NET.Base](Guilded_NET_Base#Guilded_NET_Base 'Guilded.NET.Base').[GuildedWebsocketException](GuildedWebsocketException 'Guilded.NET.Base.GuildedWebsocketException')
## GuildedWebsocketException(string, Exception) Constructor

Creates a new instance of [GuildedWebsocketException](GuildedWebsocketException 'Guilded.NET.Base.GuildedWebsocketException') with a message received.
```csharp
public GuildedWebsocketException(string message, System.Exception inner);
```

#### Parameters

<a name='Guilded_NET_Base_GuildedWebsocketException_GuildedWebsocketException(string_System_Exception)_message'></a>
`message` [System.String](https://docs.microsoft.com/en-us/dotnet/api/System.String 'System.String')

The message that was received from Guilded Websocket

<a name='Guilded_NET_Base_GuildedWebsocketException_GuildedWebsocketException(string_System_Exception)_inner'></a>
`inner` [System.Exception](https://docs.microsoft.com/en-us/dotnet/api/System.Exception 'System.Exception')

The inner exception of this error